.github/workflows/ci.yaml

```diff
@@ -0,0 +1,49 @@
+# This is a Github Workflow that runs tests on any push or pull request.
+name: CI
+on: [push, pull_request]
+jobs:
+  test:
+    if: github.event_name == 'push'
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: 14
+
+      - name: npm install
+        run: npm i
+
+      - name: Run unit tests
+        run: npm test
+
+  test_pr:
+    if: github.event_name == 'pull_request'
```

Comment:
Why so verbose, in #73 much simpler.

Reply:
1) named steps are much easier to recognize in gh-actions logs. It doesn’t matter until everything is fine)
2) checkout with depth 0 does not fetch extra data. Just the latest commit.
3) build matrix for PR verifies that code is still working on all actual LTS versions of Node.js.